src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -112,308 +118,241 @@ export interface ShippingProps {
   isCommittingMutation: boolean
 }
 
-export const ShippingRoute: FC<ShippingProps> = props => {
-  const { trackEvent } = useTracking()
-  const { relayEnvironment } = useSystemContext()
+const getOrderFulfillmentDetails = (
```

Comment:
This massive diff is going to be easier to review once all the commented out code and unused imports are gone, but personally I don't think it is worth the effort to try to understand the logic. Instead I would pay attention to the new changes here (minus commented old code) and rely on tests and observed behavior.